apps/selvbetjening-ui/src/common/dato.ts

```diff
@@ -0,0 +1,11 @@
+export const erMellomOktoberogDesember = (): boolean => {
+    const idag = new Date()
+    idag.setHours(12, 0, 0, 0)
+
+    const sisteDagIDesember = new Date(idag.getFullYear(), 11, 31)
+    sisteDagIDesember.setHours(23, 59, 59, 0)
+
+    const foersteDagIOktober = new Date(idag.getFullYear(), 9, 1)
+
+    return foersteDagIOktober <= idag && idag <= sisteDagIDesember
```

Comment:
Vil dette gi samme effekt med mindre og mer lesbar kode?
```suggestion
    const nåværendeMåned = new Date().getMonth()
    const oktoberMåned = 9
    const desemberMåned = 11

    return oktoberMåned <= nåværendeMåned && nåværendeMåned <= desemberMåned
```